### ISSUE TITLE
Call to a member function getTranslatedAttribute() on null

### ISSUE THREAD

**Issue by Sanswan** [pvtl/voyager-frontend#68]
when i wanne load the view i get this error

**Comment by Liam-PVTL** [comment]
Hi @Sanswan. Please update Voyager-page-blocks to 0.8.12 - if the problem persists could you provide a stack trace? 

I'm closing this for now as it's related to another issue, but will be happy to re-open if the problem persists. NOTE: We are not supporting translation support currently but this will stop the error from happening.

**Comment by Sanswan** [comment]
Hi Liam-PVTL, The error is not gone! i have use the new releases but stil get a error.

Symfony\Component\Debug\Exception\FatalThrowableError
Call to a member function getTranslatedAttribute() on null

http://*********************/admin/enquiries?filter=contains&key=id&s=2



TCG\Voyager\Http\Controllers\VoyagerBaseController::index
:55
vendor/tcg/voyager/src/Http/Controllers/VoyagerBaseController.php:55

On there Github page the just make a change on this line:
$displayName = $dataRow->where('field', $value)->first()->getTranslatedAttribute('display_name');

Butt i still get the error.
What am i doing wrong?

thanks

**Comment by howdu** [comment]
This error occurs when the database table has been alerted and the BREAD data is not in sync. 

Simple fix go to the BREAD and re-save. 

Code fix separate the code into a IF statement: 
```php
$dataRowField = $dataRow->where('field', $value)->first();
// Check field exists encase BREAD is not in sync
if ($dataRowField) {
	$displayName = $dataRowField->getTranslatedAttribute('display_name');
	$searchNames[$value] = $displayName ?: ucwords(str_replace('_', ' ', $value));
}
```
https://github.com/the-control-group/voyager/commit/e852148117e4c92ecb737ab1e21debfb355ad772

**Comment by Sanswan** [comment]
@howdu Thanks for your comment, i have done that and it working better now but no i have a issue with : mb_strlen() expects parameter 1 to be string, array given in the voyager browse.blade.php on line 185. on this moment i am trying to fix that issue :)

**Comment by Liam-PVTL** [comment]
@Sanswan  that issue was fixed in voyager-page-blocks to 0.8.13 I believe. Please open a separate issue in whichever repo is most relevant for further bug reports. Many thanks @howdu for saving us time investigating that issue.
